src/display.c

```diff
@@ -331,14 +331,18 @@ PyImaging_GrabScreenWin32(PyObject *self, PyObject *args) {
     HMODULE user32;
     Func_SetThreadDpiAwarenessContext SetThreadDpiAwarenessContext_function;
 
-    if (!PyArg_ParseTuple(args, "|ii", &includeLayeredWindows, &all_screens)) {
+    if (!PyArg_ParseTuple(
+            args, "|ii" F_HANDLE, &includeLayeredWindows, &screens, &screen
```

Comment:
This seems to be stealing a reference to the HDC and then deleting it, which could violate the `DeleteDC` documentation, https://learn.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-deletedc#remarks:

> An application must not delete a DC whose handle was obtained by calling the [GetDC](https://learn.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-getdc) function. Instead, it must call the [ReleaseDC](https://learn.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-releasedc) function to free the DC.

It would also be more intuitive to accept a HWND argument and get the HDC with the `GetDC` function or the [`GetWindowDC` function](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowdc) (making sure to release it with `ReleaseDC` as stated in the documentation).

Reply:
Ok, I've updated the commit to receive a HWND and use `ReleaseDC`.